app/views/workspaces/_new_modal.html.erb

```diff
@@ -0,0 +1,17 @@
+      <% if can? :create, Workspace %>
+        <%= button_tag 'Workspace', class: button_classes, data: { 'bs-toggle': 'modal', 'bs-target': '#workspaceModal', action: 'show-modal#copyFormTemplate' } %>
+        <template>
+          <%= form_with(model: Workspace.new, namespace: "new", class: 'me-2 workspace-form') do |form| %>
```

Comment:
Could we use the current project (if it exists) to set the initial project? I'm not sure how feasible that is here...

Reply:
@cbeer I'll look into it, sure.

Reply:
@cbeer I did the thing. Does this approach look OK? I tested it in a few different scenarios and it seemed to do the right thing each time.